recipes/lua-5.1/meta.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: lua
+  version: 5.1.5
+
+source:
+  fn: lua-5.1.5.tar.gz
+  url: http://www.lua.org/ftp/lua-5.1.5.tar.gz
+  md5: 2e115fe26e435e33b0d5c022e4490567
+  patches:
+    - update.patch  # [not win]
+
+build:
+  number: 0
+  detect_binary_files_with_prefix: true
+  skip: True  # [win]
```

Comment:
Is it possible to build lua on Windows? If so please open an issue in the feedstock as soon as we merge so we can keep track of this.